packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-preview.tsx

```diff
@@ -20,9 +22,11 @@ const containerStyles = css({
 });
 
 const previewHeaderStyles = css({
-  paddingTop: spacing[3],
-  paddingBottom: spacing[3],
-  paddingLeft: spacing[3],
+  padding: spacing[3],
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
```

Comment:
I will really never stop suggesting moving things to the end of the flex parent with margin instead because it's a more predictable behavior when somebody adds another element in the flex container 😄

Reply:
something like [this](https://github.com/mongodb-js/compass/pull/3651/files#diff-c7c6e400cf3d002df3f2b8665c059d582d1e5f5a592fb6dfa0864424c7b156aaR47)?